### PR TITLE
Enhance MP-DCCP concept section

### PR DESCRIPTION
**Pull request by NathalieRM**
To solve issue #44

### CODE REVIEW

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 1]
```diff
@@ -288,6 +291,34 @@ Address A1    Address A2             Address B1    Address B2
...
+      addresses at hosts.  Combinations of these multiple addresses
+      equate to the additional paths.  In the example, other potential
+      paths that could be set up are A1<->B2 and A2<->B2.  Although this
+      additional session is shown as being initiated from A2, it could
+      equally have been initiated from B1 or B2.
+
+   *  The discovery and setup of additional subflows will be achieved
+      through a path management method;
+
+   *  MPDCCP adds connection-level sequence numbers to enabke optional
```
Comment (markusa) [inline draft-ietf-tsvwg-multipath-dccp.mkd:317]:
```suggestion
   *  MPDCCP adds connection-level sequence numbers to enable optional
```

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 2]
```diff
@@ -312,13 +312,15 @@ Address A1    Address A2             Address B1    Address B2
       equally have been initiated from B1 or B2.
 
    *  The discovery and setup of additional subflows will be achieved
-      through a path management method;
+      through a path management method, whose definition is out of the
+      scope of this document.
```
Comment (abrunstrom) [inline draft-ietf-tsvwg-multipath-dccp.mkd:316]: This document does specify how to add new subflows, so this text needs to be clarified. So it specifies how to add additional subflows, but not when (under what conditions). See the MPTCP RFC for comparison.

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 3]
```diff
@@ -312,13 +312,15 @@ Address A1    Address A2             Address B1    Address B2
...
-   *  MPDCCP adds connection-level sequence numbers to enable optional
-      reordering functionalities.
+   *  MPDCCP adds connection-level sequence numbers and exchange of
+      RTT information to enable optional reordering functionalities.
 
-   *  The MPDCCP connection and their corresponding subflows are terminated as
-      regular DCCP connections, as described in ({{RFC4043}} section 8.3).
+   *  Subflows are terminated as regular DCCP connections, as described
+      in ({{RFC4340}} section 8.3). The MPDCCP connection is terminated by a
+      connection-level Close request.
```
Comment (abrunstrom) [inline draft-ietf-tsvwg-multipath-dccp.mkd:323]: Should it be a Close, close or Reset request?

**Review by abrunstrom — CHANGES_REQUESTED** [review]
One change request and one question.

File: draft-ietf-tsvwg-multipath-dccp.mkd (hunk 3 above)
Comment (NathalieRM) [reply draft-ietf-tsvwg-multipath-dccp.mkd]: I changed it to "connection-level DCCP-CloseReq or DCCP-Close message". The idea is that this message carries the MP-CLOSE option.

Comment (markusa) [reply draft-ietf-tsvwg-multipath-dccp.mkd]: Solved in https://github.com/markusa/ietf-multipath-dccp/pull/66/commits/2bc5ddb63814b6421035d7d7101dff4bfbc30848

**Pull request merged**